demo/requirements.txt

```diff
@@ -1,12 +1,14 @@
-django==2.2.13
```

Comment:
hmm, why django has been removed ?

Reply:
Because it collides with Django dependency defined in tox.ini

Reply:
Oh, it's the demo, never mind then, sorry my bad.